atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/atomic/ConsensusForgettingStore.java

```diff
@@ -33,15 +33,16 @@
  *   value until a put occurs.
  */
 public interface ConsensusForgettingStore {
+
     /**
      * An atomic put unless exists operation. If this method throws an exception, there are no consistency guarantees:
      *   1. A subsequent PUE may succeed or fail non-deterministically
      *   2. A subsequent get may return Optional.of(value), Optional.empty(), or even Optional.of(other_value) if
      *   another PUE has failed in the past non-deterministically
      */
-    void putUnlessExists(Cell cell, byte[] value) throws KeyAlreadyExistsException;
+    void atomicUpdate(Cell cell, byte[] value) throws KeyAlreadyExistsException;
 
-    void putUnlessExists(Map<Cell, byte[]> values) throws KeyAlreadyExistsException;
```

Comment:
Probably want a quick bit here on how the atomicity of this operation looks as compared to the single cell version